Xwt/Xwt/Popover.cs

```diff
@@ -84,7 +85,17 @@ public Popover (Widget content)
 			VerifyConstructorCall (this);
 			Content = content;
 		}
-		
+
+		Accessible accessible;
```

Comment:
fields at the top of the file

Reply:
I just copied here what's in Widget for `Accessible`. But let me update both of them.